charts/plex-media-server/templates/statefulset.yaml

```
@@ -119,6 +119,13 @@ spec:
         - name: {{ $key }}
           value: {{ $value | quote }}
         {{- end }}
+        {{- if and .Values.pms.claimSecret.name .Values.pms.claimSecret.value }}
+        - name: PLEX_CLAIM
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.pms.claimSecret.name }}
+              key: {{ .Values.pms.claimSecret.value }}
```

nit: I think `key` would be a better name for this, for consistency's sake.

```suggestion
              key: {{ .Values.pms.claimSecret.key }}
```